src/app/containers/ConsentBanner/index.amp.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import styled from 'styled-components';
+import Banner from './Banner/index.amp';
+
+const parentId = 'consent';
+const promptId = 'consent-prompt';
+const privacyId = 'privacy';
+const cookieId = 'cookie';
+
+const ampGeoData = {
+  ISOCountryGroups: {
+    eu: [
+      'at',
+      'be',
+      'bg',
+      'cy',
+      'cz',
+      'de',
+      'dk',
+      'ee',
+      'es',
+      'fi',
+      'fr',
+      'gb',
+      'gr',
+      'hr',
+      'hu',
+      'ie',
+      'is',
+      'it',
+      'li',
+      'lt',
+      'lu',
+      'lv',
+      'mt',
+      'nl',
+      'no',
+      'pl',
+      'pt',
+      'ro',
+      'se',
+      'si',
+      'sl',
```

Comment:
What's the source of this list of country codes? Has it been checked against other lists that we maintain (e.g. in Mozart)? Presumably it will need updating if (let's say) any countries left or joined the EU?

Also, any ideas how we can have confidence that `amp-geo` is doing what we expect it to here?

Reply:
This was copied from https://github.com/bbc/ws-alephamp-queue-renderer/blob/549bace053db39e2902c518089c58179d5328343/src/components/lib/templates/partials/geolocation.mustache

As for the confidence, We have as much confidence as we can. It follows the recomended amp implementation, after that were in the hands of the AMP gods

Reply:
Have raised https://github.com/bbc/simorgh/issues/1465